ext/dom/php_dom.stub.php

```diff
@@ -1252,7 +1252,7 @@ public function count(): int {}
         public function getIterator(): \Iterator {}
     }
 
-    class DTDNamedNodeMap implements \IteratorAggregate, \Countable
+    class DtdNamedNodeMap implements \IteratorAggregate, \Countable
```

Comment:
I saw a few properties using discouraged names, like `namespaceURI`. Can these be renamed?

Reply:
Same thing here: namespaceURI is defined per spec with that capitalization.